packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -27,45 +30,137 @@ export interface TileParams extends DocParams {
     content?: Record<string, unknown>;
 }
 
+/**
+ * Arguments used to generate the metadata for Tile documents
+ */
+export interface TileMetadataArgs {
+  controllers?: Array<string>
+  family?: string
+  schema?: CommitID | string
+  tags?: Array<string>
+  deterministic?: boolean
```

Comment:
We still need to make a final decision on how developers should specify `deterministic`. I'm a little less opposed to putting it into `DocOpts` than I was initially.  It would just mean splitting `DocOpts` up to be doctype-specific, but that's likely something we'll want to do eventually anyway.  I personally still like it here in metadata, even though it's not quite 1-1 to what comes back when metadata is loaded.  But if others think this would be better in `DocOpts` I could possibly be swayed.